chat/src/main/java/io/skygear/plugins/chat/ui/ConversationFragment.kt

```diff
@@ -235,10 +246,12 @@ open class ConversationFragment() :
 
                         override fun onGetCachedResult(messages: MutableList<io.skygear.plugins.chat.Message>?) {
                             // TODO (Steven-Chan): handle cache result to update message list
+                            callMessageFetchSuccessListener(messages)
```

Comment:
Open to discuss, should we call `onMessageFetchSuccess` on `onGetCachedResult`, or it should be on server `onSucc`? Or we can have one more event for the cache fetch?